model_compression_toolkit/quantizers_infrastructure/keras/quantize_wrapper.py

```diff
@@ -57,51 +52,104 @@ def _weight_name(name: str) -> str:
     class KerasQuantizationWrapper(tf.keras.layers.Wrapper):
         def __init__(self,
                      layer,
-                     dispatcher: NodeQuantizationDispatcher,
+                     weight_quantizers: Dict[str, BaseInferableQuantizer] = None,
```

Comment:
Why weights quantizers are a dictionary and activation are a list? can we have different quantizers for different weights of the same layer?

Reply:
yes